styles/styles.css

```diff
@@ -313,12 +500,35 @@ body {
 
 /* page-control */
 
+.page-control {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+/* !!!Временное решение, удалю, когда пропишем стили для класса visually-hidden */
+.page-control h2 {
+  display: none;
```

Comment:
https://htmlacademy.ru/blog/boost/frontend/short-12#:~:text=%D0%A1%D0%B0%D0%BC%D1%8B%D0%B9%20%D0%BF%D1%80%D0%BE%D1%81%D1%82%D0%BE%D0%B9%20%D0%B8%20%D0%BF%D0%BE%D0%BF%D1%83%D0%BB%D1%8F%D1%80%D0%BD%D1%8B%D0%B9%20%E2%80%94%20display,%D0%9F%D1%80%D0%BE%D1%81%D1%82%D0%BE%20%D0%BD%D0%B5%D1%82%20%D0%B8%20%D0%B2%D1%81%D1%91.